cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -306,6 +333,67 @@ message AvailablePackageDetail {
   google.protobuf.Any custom_detail = 13;
 }
 
+// InstalledPackageSummary
+//
+// An InstalledPackageSummary provides a summary of an installed package
+// useful when aggregating many installed packages.
+message InstalledPackageSummary {
+  // TODO: add example for API docs
+  // option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
+  //   example: '{}'
+  // };
+
+  // InstalledPackageReference
+  //
+  // A reference uniquely identifying the package.
+  InstalledPackageReference installed_package_ref = 1;
+
+  // Name
+  //
+  // A name given to the installation of the package (eg. "my-postgresql-for-testing").
+  string name = 2;
+
+  // PkgVersionReference
+  //
+  // The package version reference defines a version or constraint limiting
+  // matching package versions.
+  VersionReference pkg_version_reference = 3;
+
+  // CurrentPkgVersion
+  //
+  // The version of the package which is currently installed.
+  string current_pkg_version = 4;
+
+  // Installed package icon URL
+  //
+  // A url for an icon.
+  string icon_url = 5;
+
+  // PackageDisplayName
+  //
+  // The package name as displayed to users (provided by the package, eg. "PostgreSQL")
+  string pkg_display_name = 6;
+
+  // ShortDescription
+  //
+  // A short description of the package (provided by the package)
+  string short_description = 7;
```

Comment:
flux helmrelease doesn't provide these, so I'll need to pull these 2 off the corresponding helmchart? Just making sure

Reply:
Yes.